tests/libvmi/operations.go

```diff
@@ -0,0 +1,31 @@
+package libvmi
+
+import (
+	. "github.com/onsi/gomega"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+)
+
+func CreateVmi(client kubecli.KubevirtClient, namespace string, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
```

Comment:
You do not need to pass in the client, you can just fetch it here, it is a singletone anyway. It makes the API nicer IMO.
Not a must.